### DISCUSSION TITLE
[Development] Drop DB and create new + migrate

### DISCUSSION THREAD

**Discussion by LavissaWoW** [inventree/InvenTree#4853] [Q&A]
Working on apps where others are creating migrations that come in with merging master creates a small headache to patch up the migrations. After sorting file names and dependencies, just resetting DB state would be nice.

However, I can't seem to find an easy way to drop my entire db via `invoke` or `manage.py` in order to re-run all migrations and repopulate the db with the demo dataset. 
And tips?

**Comment by wolflu05** [comment]
Are you using the devcontainer? Deleting the db and media folder in dev manually should do the trick. Then run the setup dev task like described in the devcontainer docs.

**Reply by LavissaWoW** [reply]
Haven't made the switch yet, no. Will get around to it when I'm done with the current PR I'm working on.
Commands would be nice though, in order to not fuck up file deletion.
